### DISCUSSION TITLE
Error While Roaming Around with Map Open

### DISCUSSION THREAD

**Discussion by FuRiouSOne** [SFX-WoW/Masque#171] [Issues] [answered]
1x [ADDON_ACTION_BLOCKED] AddOn 'Masque' tried to call the protected function '<unnamed>:Hide()'.
!BugGrabber\BugGrabber.lua:573: in function <!BugGrabber\BugGrabber.lua:573>
[C]: in function `Hide'
FrameXML\WorldMapActionButton.lua:58: in function `Clear'
FrameXML\WorldMapActionButton.lua:67: in function `Refresh'
FrameXML\WorldMapActionButton.lua:19: in function `SetMapAreaID'
FrameXML\WorldMapFrame.lua:438: in function <FrameXML\WorldMapFrame.lua:401>
[C]: ?
[C]: in function `Hide'
FrameXML\UIParent.lua:2425: in function `SetUIPanel'
FrameXML\UIParent.lua:2452: in function `MoveUIPanel'
FrameXML\UIParent.lua:2483: in function `HideUIPanel'
FrameXML\UIParent.lua:2155: in function <FrameXML\UIParent.lua:2147>
[C]: in function `SetAttribute'
FrameXML\UIParent.lua:2951: in function <FrameXML\UIParent.lua:2938>
[C]: in function `HideUIPanel'
FrameXML\UIParent.lua:2917: in function `ToggleFrame'
FrameXML\WorldMapFrame.lua:248: in function `ToggleWorldMap'
[string "TOGGLEWORLDMAP"]:1: in function <[string "TOGGLEWORLDMAP"]:1>

Locals:
InCombatSkipped

**Comment by StormFX** [comment, marked as answer]
This is taint from an add-on opening/hiding the world map/quest journal from code.
